packages/purgecss-webpack-plugin/README.md

```diff
@@ -136,6 +136,10 @@ new PurgeCSSPlugin({
 
 If `true` all removed selectors are added to the [Stats Data](https://webpack.js.org/api/stats/) as `purged`.
 
+* #### rejectedCss
```

Comment:
Not sure how complicated it would be, but I imagine it would be more webpack friendly if it uses their [template string](https://webpack.js.org/configuration/output/#template-strings) configuration options.

So it would allow for something like:
```ts
const options = {
  output: '[name].rejected.[ext]'
}
```